src/open_inwoner/templates/pages/profile/contacts/list.html

```diff
@@ -11,46 +11,53 @@ <h1 class="h1" id="title">
 <!--List of contact requests: receiver's view-->
 {% if pending_approvals %}
     {% with pending_approvals.count as approvals_count %}
-    <div class="approval{% if approvals_count == 1 %} approval--solo{% endif %}">
-        <div class="approval__header">
-            <h2 class="h2">{% trans "U bent toegevoegd als contactpersoon" %}</h2>
-            <p class="p">{% trans "By accepting you agree on sharing your personal data (first name, last name and phonenumber)." %}</p>
+    <div class="approval-container{% if approvals_count == 1 %}--solo{% endif %}">
+        <div class="approval{% if approvals_count == 1 %} approval--solo{% endif %}">
```

Comment:
I noticed a double `approval--solo` if/else here but I don't know if it is a thing.

Reply:
The parent is `container-approval--solo` and the child `approval--solo`,  but I couldn't think of another way for having this condition in 2 different class names.

Reply:
I'd expect you could select on the parent? 

```
.approval-container .approval { /* normal rules for .approval */ }
.approval-container--solo .approval { /* rules for .approval when solo */  }
```